canvas_modules/common-canvas/src/common-properties/components/truncated-content-tooltip/truncated-content-tooltip.scss

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2021 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.properties-truncated-tooltip {
+	overflow: hidden;
+	.tooltip-trigger {
+		width: inherit;
+		overflow: hidden;
+		text-overflow: ellipsis;
+		white-space: nowrap;
+		span {
```

Comment:
Why was this moved from readonly.scss?  Seem odd to have that here. I'm only referencing the `span` piece.

Reply:
CSS under `span` is needed only for readonly control. Moved `span` into readonly.scss file.